packages/web/src/serviceWorker.ts

```diff
@@ -11,9 +12,9 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
-  window.location.hostname === 'localhost' ||
+  window.location.hostname === "localhost" ||
```

Comment:
I'd prefer single quotes in scripts files. Let's discuss this with a team

Reply:
Done.

Reply:
This should've been enforced by the eslint, no?

Reply:
Also I don't think we really need serviceWorker file at all anyway :)